src/components/Breadcrumb/Breadcrumb.story.jsx

```diff
@@ -106,11 +106,15 @@ storiesOf('Watson IoT | Breadcrumb', module)
               <BreadcrumbItem href="#" title="3 A really long page name">
                 3 A really long page name
               </BreadcrumbItem>
-              <BreadcrumbItem href="#" title="4 Another page">
-                4 Another page
+              <BreadcrumbItem href="#" title={text('Breadcrumb 4 text', '4 Another page')}>
+                {text('Breadcrumb 4 text', '4 Another page')}
               </BreadcrumbItem>
-              <BreadcrumbItem href="#" isCurrentPage title="5th level page">
-                5th level page
+              <BreadcrumbItem
+                href="#"
+                isCurrentPage
+                title={text('Breadcrumb 5 text', '5th level page')}
+              >
+                {text('Breadcrumb 5 text', '5th level page')}
```

Comment:
I don't see this knob, only the knob at line 115. I'm guessing its because they have the same title.

Reply:
it is supposed to be the same knob. This populates the title with the same value

Reply:
![image](https://user-images.githubusercontent.com/1590966/75361439-4a613280-5885-11ea-9237-5e716abb345f.png)